client/src/components/ContentNavigator/index.ts

```diff
@@ -247,6 +248,16 @@ class ContentNavigator implements SubscriptionProvider {
           "workbench.actions.treeView.sas-content-navigator.collapseAll"
         );
       }),
+      workspace.onDidChangeConfiguration(
+        async (event: ConfigurationChangeEvent) => {
+          if (event.affectsConfiguration("SAS.connectionProfiles")) {
```

Comment:
nit: just thinking perhaps this can be extracted to be our own event or API for example `onProfilesChange` so that each component won't write this line in each place. But it can definitely be in future PR separately.

Reply:
I'd definitely advocate for a separate pr after the 0.1.3 release, but this seems like it could be an improvement